CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
```

Comment:
Reinforcing what @matheusps said, you don't need to add changelog. Please remove these additions and add #trivial to your PR description. Thank you!